### DISCUSSION TITLE
Is there a way to get exact dates per matchup for fantasy basketball?

### DISCUSSION THREAD

**Discussion by tim7398** [cwendt94/espn-api#441] [ESPN API] [answered]
The matchup and box score classes do not currently expose the current matchup dates, but on ESPN they are able to display a date range per matchup. During the regular season, we can infer a matchup week goes from monday to sunday, but for playoffs, this assumption is no longer valid. 

Is there a way to utilize the current API to dynamically fetch these dates?

**Comment by cwendt94** [comment, marked as answer]
Thats a great idea. I will have to check to see if those dates are returned in the box score api call or if I would need to get it from another endpoint. But I think either way I should be able to get that data.

**Reply by martincham** [reply]
Was there any progress on this? I only see matchup IDs, but not dates in the data returned from the get_league request.

**Reply by cwendt94** [reply]
I still haven't been able to find where they get the Dates. I have not been able to find it in any of their API calls.
